myip.go

```diff
@@ -49,3 +50,16 @@ func GetIP() (net.IP, error) {
 
 	return nil, fmt.Errorf("Unable to find IP")
 }
+
+// ExternalIP returns your public IP.
```

Comment:
This should return net.IP instead of a string. See the other function as an example.